### DISCUSSION TITLE
How should we comment on TPI Document ?

### DISCUSSION THREAD

**Discussion by smartopian** [KantaraInitiative/ancr-wg#6] [Q&A] [answered]
I am  looking for the prcocess for putting the TPI doc together for comment, which is currently now on the wiki. 

Previous process has been to put in a google doc and collaborate form there, where the ip is unprotected by the ANCR License,. 
Then to put it into a Kantara Template, and ask for comments, via a comment sheet.

@arminpkathrein  do you know of a better way?  or a way we can use github? 
@IDmachines do you recommend a process here ?

**Comment by IDmachines** [comment, marked as answer]
You can upload an odt to github and then track it from there, I believe.

 

From: Mark Lizar ***@***.***> 
Sent: Tuesday, August 8, 2023 2:06 PM
To: KantaraInitiative/WG-ANCR ***@***.***>
Cc: Salvatore D'Agostino ***@***.***>; Mention ***@***.***>
Subject: [KantaraInitiative/WG-ANCR] How should we comment on TPI Document ? (Discussion #6)

 

I am looking for the prcocess for putting the TPI doc together for comment, which is currently now on the wiki.

Previous process has been to put in a google doc and collaborate form there, where the ip is unprotected by the ANCR License,.
Then to put it into a Kantara Template, and ask for comments, via a comment sheet.

@arminpkathrein <https://github.com/arminpkathrein>  do you know of a better way? or a way we can use github?
@IDmachines <https://github.com/IDmachines>  do you recommend a process here ?

**Comment by smartopian** [comment]
@IDmachines @GigiAgassini  i have tried making a Markdown version of the TPI spec as oppose to ODT.  
 
 the real issue what way do we take comments on it., ?   if its by line number  - using a comment sheet - then we can use Markdown - but if its online, then Teams and MS office would be used - as tools not in compliance with our framework.  ?

**Reply by smartopian** [reply]
Here[ is the link](https://github.com/KantaraInitiative/WG-ANCR/blob/main/TPI/TPI%20Spec.md)

**Reply by smartopian** [reply]
using git to check in and out the TPI specification  as a word doc, which is also usable for in document collaboration
